tabpfn_client/client.py

```diff
@@ -275,8 +277,16 @@ def register(
 
         response = self.httpx_client.post(
             self.server_endpoints.register.path,
-            params={"email": email, "password": password, "password_confirm": password_confirm,
-                    "validation_link": validation_link}
+            params={
+                "email": email, 
+                "password": password, 
+                "password_confirm": password_confirm,
+                "validation_link": validation_link, 
+                "company": additional_info["company"], 
```

Comment:
i think the better thing to use here is `**additional_info` if you do not have an argument against that

Reply:
Changed to this **additional_info and it is working fine as expected.